webpack/assets/javascripts/react_app/components/BreadcrumbBar/BreadcrumbBar.js

```diff
@@ -7,6 +7,13 @@ import Breadcrumb from './components/Breadcrumb';
 import './breadcrumbswitcher.scss';
 
 class BreadcrumbBar extends React.Component {
+  componentDidMount() {
+    const { data, runTour } = this.props;
+    if (data.breadcrumbItems.length > 1) {
+      runTour({ delay: false });
```

Comment:
I think it's bad UX to start the tour without user opt-in.  Is this PR meant to be merged?  If so, maybe we need to find a good way to have the user opt-in to tours.